litmus/director/openebs-install/test.yml

```diff
@@ -0,0 +1,171 @@
+---
+- hosts: localhost
+  connection: local
+
+  vars_files:
+    - test_vars.yml
+
+  tasks:
+  
+    ## Generating the testname for deployment
+    - include_tasks: /ansible-utils/create_testname.yml
+
+    ## RECORD START-OF-TEST IN LITMUS RESULT CR
+    - include_tasks: /ansible-utils/update_litmus_result_resource.yml
+      vars:
+        status: 'SOT'
+      
+    - set_fact:
+        director_url : "http://{{ director_ip }}:30380"
+
+    ## Getting the username
+    - name: Get username
+      shell: cat /etc/secret-volume/username
+      register: username
+
+    ## Getting the password     
+    - name: Get password
+      shell: cat /etc/secret-volume/password
+      register: password
+    
+    - name: Fetch the project id of the cluster
+      uri:
+        url: "{{ director_url }}/v3/groups/{{ group_id }}/project"
+        method: GET
+        url_username: "{{ username.stdout }}"
+        url_password: "{{ password.stdout }}"
+        force_basic_auth: yes
+        return_content: yes
+        status_code: 200
+      register: project_details
+
+    ## Fetching project id
+    - name: Fetching project id
+      set_fact:
+        project_id: "{{ project_details.json.data[0].id }}"
+    
+    ## Checking the node of connected Cluster
+    - name: Get into node of the cluster
+      uri:
+        url: "{{ director_url }}/v3/groups/{{ group_id }}/clusters/{{ cluster_id }}/nodes"
+        method: GET
+        url_username: "{{ username }}"
+        url_password: "{{ password }}"
+        force_basic_auth: yes
+        return_content: yes
+        body_format: json
+        status_code: 200,202
+      register: node_cluster
+    
+    ## Getting node-1 id.
+    - name: get node-1 id
```

Comment:
Start the task name with a capital letter. This is just to ensure that we follow a similar convention all across the tasks.